R/azure_storage_setup.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
```

Comment:
I realize this script may work on your system, but I foresee problems. Configuration for Azure is out of scope, `targets` assumes you went through the configuration and setup process separately. I would prefer to instead link to a setup script where it exists in the documentation of Azure or the R package interfaces. And only R scripts can go in the `R/` folder of the package.

Reply:
My intention for this script is documentation or dev notes. It should be helpful for someone who does not have a Storage Account or Service Principal to use with `AzureStor`. It does not contain any variables or logic necessary to use the utility functions — a user should be able to bring their own account to the party.

As mentioned below, I do think you are correct and there is likely a more standard way to set everything up. I'll have a another pass to see what is possible, with the aim of stripping out the setup script from the final feature.